From c74cce202a87e55ee7693a07bda6e417929bc2fc Mon Sep 17 00:00:00 2001
From: Vishal Mahaveer <vishalm@ti.com>
Date: Fri, 10 Aug 2012 08:13:40 -0500
Subject: [PATCH] Android.mk: update version string

Update version string to differentiate this tree from Google tree

Signed-off-by: Vishal Mahaveer <vishalm@ti.com>
---
 Android.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Android.mk b/Android.mk
index 77d72ea..90236f2 100644
--- a/Android.mk
+++ b/Android.mk
@@ -1,3 +1,3 @@
-ifeq ($(WPA_SUPPLICANT_VERSION),VER_0_8_X)
+ifeq ($(WPA_SUPPLICANT_VERSION),VER_0_8_X_TI)
     include $(call all-subdir-makefiles)
 endif
-- 
1.7.11.1

